### DISCUSSION TITLE
What is the wheel rotation unit?

### DISCUSSION THREAD

**Discussion by Sci-fer** [UPBGE/upbge#1898] [Resources for UPBGE 0.3+]
The documentation doesn't specify the units returned by [getWheelRotation](https://upbge.org/#/documentation/docs/latest/api/bge.types.KX_VehicleWrapper.html#bge.types.KX_VehicleWrapper.getWheelRotation).
Is it radians, revolutions or what?

Kind regards

**Comment by BlenderVdb06** [comment]
Radian 
radian convertion
3.14/180

create another Math Node type multypli 
yours degrée want convert multipli radian convert
![image](https://github.com/user-attachments/assets/b30b44f9-e1a7-49d0-8961-3c3f1dd43c2c)
